dask/array/gufunc.py

```diff
@@ -382,7 +382,7 @@ def apply_gufunc(
 
     ## Axes: transpose input arguments
     transposed_args = []
-    for arg, iax, input_coredims in zip(args, input_axes, input_coredimss):
+    for arg, iax, _ in zip(args, input_axes, input_coredimss):
```

Comment:
Similar comment here: if we're not using `input_coredims` then let's just remove `input_coredimss` from the `zip` iteration 

```suggestion
    for arg, iax in zip(args, input_axes):
```